boa/environment.py

```diff
@@ -311,9 +311,22 @@ def _hook_trace_computation(self, computation, contract=None):
             child_contract = self._lookup_contract_fast(child.msg.code_address)
             self._hook_trace_computation(child, child_contract)
 
-    def get_code(self, address):
+    def get_code(self, address: _AddressType) -> bytes:
         return self.evm.get_code(Address(address))
 
+    def set_code(self, address: _AddressType, code: bytes) -> None:
+        assert self.evm.is_forked, "The EVM is not forked, cannot set code"
+        self.evm.set_code(Address(address), code)
+
+    def get_storage(
+        self, address: _AddressType, slot: int, from_journal: bool = True
```

Comment:
i don't think we need `from_journal`, as it exposes py-evm internals most users are not concerned with